NSS-Day-Cohort-42/rare-API-media-moguls
# Cb get all posts

#### Changes Made
1. Added file `post.py` to `views` directory.
2. `Post.py` file contains Post Viewset to fetch all posts and individual posts by id.  
3. Added 'posts.json' to fixtures, with seeded data for posts migration
​
#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout cb-get-all-posts
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
python manage.py runserver
```
4. Initialize virtual environment, and run the server.
```
pipenv shell

```
5. Test app functionality.
> developer should be able to make migrations for, migrate, and load data for posts once categories fixtures have been added 
6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
